src/auxiliary/deprecate.jl

```diff
@@ -58,4 +35,161 @@ Base.@deprecate EuclideanProduct() EuclideanInnerProduct()
 
 Base.@deprecate insertunit(P::ProductSpace, args...; kwargs...) insertleftunit(args...; kwargs...)
 
+# Factorization structs
+@deprecate QR() MatrixAlgebraKit.LAPACK_HouseholderQR()
+@deprecate QRpos() MatrixAlgebraKit.LAPACK_HouseholderQR(; positive=true)
+
+@deprecate QL() MatrixAlgebraKit.LAPACK_HouseholderQL()
+@deprecate QLpos() MatrixAlgebraKit.LAPACK_HouseholderQL(; positive=true)
+
+@deprecate LQ() MatrixAlgebraKit.LAPACK_HouseholderLQ()
+@deprecate LQpos() MatrixAlgebraKit.LAPACK_HouseholderLQ(; positive=true)
+
+@deprecate RQ() MatrixAlgebraKit.LAPACK_HouseholderRQ()
+@deprecate RQpos() MatrixAlgebraKit.LAPACK_HouseholderRQ(; positive=true)
+
+@deprecate SDD() MatrixAlgebraKit.LAPACK_DivideAndConquer()
+@deprecate SVD() MatrixAlgebraKit.LAPACK_QRIteration()
+
+@deprecate Polar() MatrixAlgebraKit.PolarViaSVD(MatrixAlgebraKit.LAPACK_DivideAndConquer())
+
+# truncations
+const TruncationScheme = MatrixAlgebraKit.TruncationStrategy
+@deprecate truncdim(d::Int) truncrank(d)
+@deprecate truncbelow(ϵ::Real) trunctol(ϵ)
+
+# factorizations
+# --------------
+_kindof(::MatrixAlgebraKit.LAPACK_HouseholderQR) = :qr
+_kindof(::MatrixAlgebraKit.LAPACK_HouseholderLQ) = :lq
+_kindof(::MatrixAlgebraKit.LAPACK_SVDAlgorithm) = :svd
+_kindof(::MatrixAlgebraKit.PolarViaSVD) = :polar
+_kindof(::DiagonalAlgorithm) = :svd # shouldn't really matter
+
+_drop_alg(; alg=nothing, kwargs...) = kwargs
+_drop_p(; p=nothing, kwargs...) = kwargs
+
+# orthogonalization
+export leftorth, leftorth!, rightorth, rightorth!
+function leftorth(t::AbstractTensorMap, p::Index2Tuple; kwargs...)
+    Base.depwarn("`leftorth` is no longer supported, use `left_orth` instead", :leftorth)
+    return leftorth!(permutedcopy_oftype(t, factorisation_scalartype(leftorth, t), p); kwargs...)
+end
+function rightorth(t::AbstractTensorMap, p::Index2Tuple; kwargs...)
+    Base.depwarn("`rightorth` is no longer supported, use `right_orth` instead", :rightorth)
+    return rightorth!(permutedcopy_oftype(t, factorisation_scalartype(rightorth, t), p); kwargs...)
+end
+function leftorth(t::AbstractTensorMap; kwargs...)
+    Base.depwarn("`leftorth` is no longer supported, use `left_orth` instead", :leftorth)
+    return leftorth!(copy_oftype(t, factorisation_scalartype(leftorth, t)); kwargs...)
+end
+function rightorth(t::AbstractTensorMap; kwargs...)
+    Base.depwarn("`rightorth` is no longer supported, use `right_orth` instead", :rightorth)
+    return rightorth!(copy_oftype(t, factorisation_scalartype(rightorth, t)); kwargs...)
+end
+function leftorth!(t::AbstractTensorMap; kwargs...)
+    Base.depwarn("`leftorth!` is no longer supported, use `left_orth!` instead", :leftorth!)
```

Comment:
I kind of prefer `... is deprecated` over `... is no longer supported`, as the latter seems to suggest that it already doesn't work anymore, whereas you've gone to great lengths to make sure they do still work.